spinalcordtoolbox/deepseg/models.py

```diff
@@ -535,7 +535,7 @@ def list_tasks_string():
     # Display coloured output
     color = {True: 'LightGreen', False: 'LightRed'}
     table = f"{'TASK':<30s}{'DESCRIPTION':<50s}\n"
-    table += f"{'-' * 80}\n"
+    table += f"- {'-' * 76}- \n"
```

Comment:
TODO: Find a more aesthetically pleasing way of tricking Sphinx into thinking that this table header is not an rST header.

Reply:
Easy peasy! Both of the problematic tables were in argparse `epilogs`, so if we just add `:noepilog:` to both `sct_deepseg` and `sct_download_data`, we avoid the mis-rendered tables and prevent the error.